### PR TITLE
DBus Entitlement object

### PR DESCRIPTION
**Pull request by jstavel**
it offers methods `GetStatus` `GetPools`.

### CODE REVIEW

**Comment by jstavel** [comment]
This PR is not ready for verification yet.

**Comment by awood** [comment]
bot, ok to test

File: src/rhsmlib/services/entitlement/certificate_filter.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
+from __future__ import print_function, division, absolute_import
```
Comment (kahowell) [inline src/rhsmlib/services/entitlement/certificate_filter.py]: Please leave the contents in `src/subscription_manager/utils.py`.

File: src/rhsmlib/services/entitlement/merged_pools.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
+from __future__ import print_function, division, absolute_import
```
Comment (kahowell) [inline src/rhsmlib/services/entitlement/merged_pools.py]: This should just stay in `src/subscription_manager/managerlib.py` and be imported.

File: src/rhsmlib/services/entitlement/methods.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
+from __future__ import print_function, division, absolute_import
```
Comment (kahowell) [inline src/rhsmlib/services/entitlement/methods.py]: The classes and methods in this file should just be imported.

File: src/rhsmlib/services/entitlement/pool_filter.py [hunk 4]
```diff
@@ -0,0 +1,183 @@
+from __future__ import print_function, division, absolute_import
```
Comment (kahowell) [inline src/rhsmlib/services/entitlement/pool_filter.py]: The classes and methods in this file should just be imported from managerlib.

**Comment by kahowell** [comment]
I also wanted to mention. If you want to redo a PR to make it easier to review, [interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) works nicely. I tried this and had to manually resolve some merge conflicts, but it didn't take long.

**Comment by jstavel** [comment]
This PR is too complicated to merge. And I have made a mistake with git rebase/merge. So it is easier to take a piece of useful code from my branch to new branch rather to merge this PR.

**Pull request closed without merging**